components/utilities/utilities.css

```diff
@@ -0,0 +1,50 @@
+@import url(../../css/main.css);
+.text-left{
```

Comment:
One suggestion. For readability purposes one space after selector looks good. `.text-left {`